digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-core/src/main/java/io/muenchendigital/digiwf/s3/integration/adapter/in/rest/RestExceptionMapping.java

```diff
@@ -0,0 +1,16 @@
+package io.muenchendigital.digiwf.s3.integration.adapter.in.rest;
+
+import io.muenchendigital.digiwf.s3.integration.application.port.in.FileExistenceException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionMapping {
+
+  @ExceptionHandler
+  public ResponseEntity<String> handleFileExistenceException(FileExistenceException exception) {
```

Comment:
There I found one, what about the other exceptions?

Reply:
No uniform handling